commands/team_e2e_test.go

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/mattermost/mmctl/client"
+	"github.com/mattermost/mmctl/printer"
+)
+
+func (s *MmctlE2ETestSuite) TestDeleteTeamsCmdF() {
+	s.SetupTestHelper().InitBasic()
+
+	s.RunForAllClients("Error deleting team which does not exist", func(c client.Client) {
+		printer.Clean()
+		teamName := "existingName"
```

Comment:
As suggested in a previous PR, I'd set `teamName` to `nonExistentName` to make it more explicit we are looking for something that doesn't exist.

Reply:
Done.